src/content/smart-bulletins/034.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Smart Bulletin No. 034"
+intro: "Requests for JPMorgan Chase & Co. (JPM) GSA SmartPay 2® (SP2) Data"
+number: "No. 034"
+order: 034
+businessline: "All Business Lines"
+published: "Aug 28, 2020"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| August 28, 2020 | Implementation of Smart Bulletin #34 |
+
+## Effective Date
+ Effective upon issuance and shall remain in force until modified or rescinded. 
+
+## Business Line(s) Affected
+
+All Business Lines
+
+## Introduction
+
+The purpose of this Smart Bulletin is to detail the process which agencies serviced by JPM under SP2 should follow when requesting historical JPM SP2 data from the GSA Center for Charge Card Management (CCCM). This Smart Bulletin includes information on how to request JPM SP2 data from CCCM, the duration that data will be available and the expected turnaround time for data requests.
```

Comment:
The purpose of this Smart Bulletin is to detail the process which agencies serviced by JPM under SP2 should follow when requesting historical JPM SP2 data from the GSA Center for Charge Card Management (CCCM). This Smart Bulletin includes information on how to request JPM SP2 data from CCCM, the duration that data will be available, and the expected turnaround time for data requests.